### ISSUE TITLE
fetchMock.mock doesn't work as expected.

### ISSUE THREAD

**Issue by rhobot** [wheresrhys/fetch-mock-jest#9]
Hi,

I found that `fetchMock.mock` is a function instead of an object with keys like `calls` and `results`. I have found that `fetchMock.fetchHandler.mock` is what I expected.

`fetchMock.mock` looks like this

```js
function(...args) {
  setDebugPhase('setup');
  if (args.length) {
    this.addRoute(args);
  }
    
  return this._mock();
}
```

I was wondering this is expected.
Thanks for your help.

**Comment by wheresrhys** [comment]
It should still have `calls` and `results` properties though as I merge the jest mock object and the fetch-mock mock function together. Have you checked the properties don't exist?

**Comment by wheresrhys** [comment]
Fixed in 1.2.5 - thanks for the bug report
